### PR TITLE
[MNOE-106] Admin Panel Improvments

### PR DESCRIPTION
**Pull request by claire00**

### CODE REVIEW

**Comment by ouranos** [comment]
@alexnoox Can you have a look at the Angular part?
I'm looking at the ruby and fixing the specs.

File: core/app/controllers/mno_enterprise/application_controller.rb [hunk 1]
```diff
@@ -98,8 +98,12 @@ def return_to_url(resource)
     # Redirect to previous url and reset it
     def after_sign_in_path_for(resource)
       previous_url = session.delete(:previous_url)
-      url = MnoEnterprise.router.dashboard_path || main_app.root_url
-      return (return_to_url(resource) || previous_url || url)
+      default_url = if resource.respond_to?(:admin_role) && resource.admin_role.present?
+        '/admin/'
```
Comment (alexnoox) [inline core/app/controllers/mno_enterprise/application_controller.rb]: @ouranos: Can we use something like `MnoEnterprise.router.admin_path`?

**Comment by alexnoox** [comment]
LGTM

**Comment by ouranos** [comment]
@claire00 I've added the specs in 777dc88b and pushed to `feature/admin_panel_improvement`, update your local branch (`git pull upstream feature/admin_panel_improvement`) and have a look for your information.

Could you also add @alexnoox suggestion?

Thanks

**Comment by ouranos** [comment]
Closing in favor of #54 to merge in `3.1` branch

**Pull request closed without merging**